docs/why-stacked.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_label: "Why Stacked"
+sidebar_position: 0
+---
+
+# What is Stacked
+
+Stacked is the production framework for Flutter. For teams or solo developers that require production level code, Stacked makes it easy to achieve. [FilledStacks](https://www.youtube.com/filledstacks) is a mobile software development agency with experience using Native, Xamarin and now Flutter to develop over 30 applications for large companies. We know what is required to build big, **scalable**, **testable** and **maintanable applications**, this is the core focus of Stacked.
```

Comment:
I don't think stacked is a 'production` framework. How about
```
Stacked is a thin yet opionated prodictivity framework built atop Flutter to create MVCC Flutter apps. 

Stacked simplifies developing and maintaining production ready Flutter apps by 
* generating boilerplate..
* ...
* ...

 ...and much more. 
 
 [FilledStacks](https://www.youtube.com/filledstacks) is a mobile software development agency with experience using Native, Xamarin and Flutter to produce best-in-class mobile applications for large enterprises. Our experience has shaped Stacked around three central tenets: **scalability**, **testability** and **maintainability**.
 ```

Reply:
I left the production framework part, mostly because it's the word a lot of the stacked responses used to the question "what type of developer or team would benefit from stacked". 

Outside of the buzz word, why do you say it's not a production framework? Is it because of where it is now, or can it technically not be a production framework?

Reply:
I used the simplifies line without the points.

Reply:
Production framework doesn't really make sense, because all frameworks can and most are used in production. The fact that most people are using it in production just means its a good framework and people are actually using it.

Reply:
@gresnick it's true. The "production" term doesn't make sense anywhere in that way because if you're good enough you'll make anything work in production. This is more saying that "Stacked is built specifically to enforce practices that make maintaining software that's in production and used in the real world easier." The reason I compare this to nextjs a lot for flutter is because a lot of the responses of what people say about stacked that use it has the same words as on the nextjs website. Which is "The production react framework".